src/main/xerus/monstercat/api/Player.kt

```diff
@@ -169,7 +171,13 @@ object Player: FadingHBox(true, targetHeight = 25) {
 		onFx {
 			showText(text)
 			if(coverUrl != null) {
-				children.add(0, ImageView(Covers.getCoverImage(coverUrl!!, 24)))
+				val imageView = ImageView(Covers.getThumbnailImage(coverUrl!!, 24))
+				imageView.setOnMouseClicked {
+					if (it.button == MouseButton.PRIMARY && (it.clickCount == 1 || it.clickCount == 2)) {
```

Comment:
why is the clickCount relevant?

Reply:
e8858dcd7c9c1e6f0658fe55c1781fcfe0417165